Makefile

```diff
@@ -15,13 +15,13 @@ build-bundle:
 build-swagger:
 	cd swagger && npm run build
 test:
-	JEKYLL_NO_GITHUB=true bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
+	bundle exec htmlproofer ./_site --assume-extension --check-html --allow-hash-href --empty-alt-ignore --only-4xx --disable-external --url_ignore "/esQuery\.config\.js/"
 local:
-	JEKYLL_NO_GITHUB=true bundle exec jekyll serve --incremental
+	bundle exec jekyll serve --incremental
```

Comment:
`JEKYLL_NO_GITHUB=true` here is needed because it might be possible that `GITHUB_ACCESS_TOKEN ` env var is set  thus we need this to invalidate it in local build